### ISSUE TITLE
Unexpected count length error

### ISSUE THREAD

**Issue by marqgray** [Michael-wigontherun/ESLifyEverything#16]
When attempting to run the tool on [Dark Envoy - Vampire Powers](https://www.nexusmods.com/skyrimspecialedition/mods/16425) I am getting an error despite following the proper procedure. Here is the console log:

```
The output folder will still need to be set as a mod for skyrim.
BSA Connections imported from previous session.
Sucessful startup
ImportAllCompactedModData detected true. Disabling AutoRunESLify.

Generating "_ESlEverything.json" files. Ignore order of output

Starting output of Sessions
Outputing to .\CompactedForms\DarkEnvoy.esp_ESlEverything.json





Starting BSA mod connections generation.
Checking DarkEnvoy.bsa for mod connections.
DarkEnvoy.bsa cached previously.

Processed 1 out of 1 plugin BSAs






Building and Updating zMerge Data References





Importing all Mod Data
No Compacted Mod Data at S:\Living Skyrim\Stock Game\Data\CompactedForms. Safe to ignore.
Reading compacted mod data from .\CompactedForms\DarkEnvoy.esp_ESlEverything.json
DATA record had unexpected length that did not match previous counts 4 != 0

DarkEnvoy.esp_ESlEverything.json is outdated or was not saved.

DarkEnvoy.esp was not fully compacted. New forms detected. Recompact the plugin in xEdit, ensure you recompact the plugin in the output folder as well as set the esl flag.
Please delete previously edited and outputed data files and\or plugins for DarkEnvoy.esp.
Then recompact and save DarkEnvoy.esp in xEdit. Do not delete the CompactedModData file. It can still be updated.
ESLify Everything will ignore DarkEnvoy.esp compacted mod data for this run.
Press Enter to continue...
```

I loaded the esp into xEdit, compacted while ignoring the cell warnings, set esl, and then run the tool. I verified that the compactdata folder was empty before the process, as well deleting xEdit logs.

**Comment by Michael-wigontherun** [comment]
Are you on the 3.3.2 of Dark Envoy? and on 4.4.1 or newer of ESLify Everything? and is the error repeatable

**Comment by marqgray** [comment]
I'm on Eslify Everything 4.5.1 and Dark Envoy 3.3.2. Yes, error is repeatable

**Comment by Michael-wigontherun** [comment]
I can't recreate the error. 

Do you only copy the ESLifyEverything.dll from my package? I believe it is similar to a error that was fixed in version 0.40.0 of Mutagen. 

If that isn't it then try delete the Compacted Mod Data "DarkEnvoy.esp_ESlEverything.json" and recompact it without saving it again.
